### PR TITLE
Contributing.md: Make our stance on leaked information explicit

### PR DESCRIPTION
**Pull request by JosJuice**
We have been operating on this principle for a while. Let's make it explicit so that new contributors know about it.

Feedback on wording and structure is welcome.

### CODE REVIEW

**Comment by BhaaLseN** [comment]
I wonder if "confidental information about the GameCube or Wii" is good enough, or whether people can possibly see this as invitation to go like "oh, but I worked on Triforce lemme copy this over". Not sure how to extend on this though, in a way thats generic yet clear enough to get that point across. Maybe something like "about the GameCube, Wii or its associated hardware/software"?

Otherwise, the rest looks good to me; and just like yourself I was surprised that we didn't have this in writing in here yet.

**Comment by JosJuice** [comment]
I made the wording a bit broader and explicitly mentioned Triforce.

**Review by BhaaLseN — APPROVED** [review]
I don't have anything else to add at this point; so LGTM to get something out there.

File: Contributing.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+Summary:
+
+- [Trade secrets](#trade-secrets)
+- [Code licensing](#code-licensing)
+
+## <a name="trade-secrets"></a>Trade secrets
+
+Following all relevant laws is of utmost importance for an emulation project like Dolphin.
+
+If you know any confidential information related to the GameCube, Wii or Triforce, either because you signed a non-disclosure agreement or because you looked at leaked materials, we ask that you don't contribute code to Dolphin **at all**. While accepting code from contributors who know confidential information is legal if the code is unrelated to the confidential information, we refuse to accept code from such contributors because it greatly increases our review burden and increases the legal risk we take.
```
Comment (dreamsyntax) [inline Contributing.md:21]: comma after `Wii`

**Review by dreamsyntax — CHANGES_REQUESTED** [review]

**Review by mbc07 — APPROVED** [review]

**Review by AdmiralCurtiss — APPROVED** [review]

**Review by OatmealDome — APPROVED** [review]

**Review by dreamsyntax — APPROVED** [review]

**Pull request merged**